index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
+  <title>Background and Gradients</title>
+</head>
+
+<body>
+  <nav class="gradient-header">
+    <ul class="global-nav">
+      <li class="nav-items">
+        <a class="logo-a" href=""><span><img  class="apple-logo" src="../assets/apple_logo_PNG19673.png" alt=""></span></a>
```

Comment:
The logo will show up if you change the `src` attribute to `src="./assets/apple_logo_PNG19673.png"`.